src/connection.js

```diff
@@ -21,6 +21,8 @@ var Session = require('./session');
  * @extends TTransactionGateway
  */
 function Connection(id, options) {
+  Connection.validateOptions(options);
```

Comment:
Hmmm, what is the reason we have options in the first place? Not simple address?